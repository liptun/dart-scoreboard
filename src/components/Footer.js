import React from 'react'

import '../styles/footer.scss'

const Footer = () => (
    <footer className="footer">
        <p>
            Developed by{' '}
            <a href="https://github.com/liptun" target="_blank" rel="noreferrer">
                Rafa≈Ç Karczmarzyk aka LiptuN
            </a>
        </p>
    </footer>
)
export default Footer
