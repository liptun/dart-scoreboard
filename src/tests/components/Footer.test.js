import React from 'react'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import Footer from '../../components/Footer'

test('Render Footer component', () => {
    const renderer = new ReactShallowRenderer()
    renderer.render(<Footer />)
    expect(renderer.getRenderOutput()).toMatchSnapshot()
})
