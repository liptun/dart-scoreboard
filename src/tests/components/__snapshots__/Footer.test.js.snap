// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Footer component 1`] = `
<footer
  className="footer"
>
  <p>
    Developed by
     
    <a
      href="https://github.com/liptun"
      target="_blank"
    >
      LiptuN
    </a>
  </p>
</footer>
`;
